src/Language/PureScript/Errors.hs

```diff
@@ -522,6 +522,7 @@ errorSuggestion err =
           CST.WarnDeprecatedForeignKindSyntax -> suggest $ "data " <> CST.printTokens (drop 3 toks)
           CST.WarnDeprecatedKindImportSyntax -> suggest $ CST.printTokens $ drop 1 toks
           CST.WarnDeprecatedKindExportSyntax -> suggest $ CST.printTokens $ drop 1 toks
+          CST.WarnDeprecatedCaseOfOffsideSyntax -> suggest $ CST.printTokens toks
```

Comment:
I don't think this is correct. This is printing a replacement suggestion. This will end up suggesting a replacement of the first and last tokens of the expression.

Reply:
Ah, ok. I just dropped this by returning `Nothing`.